arkouda/pdarrayclass.py

```diff
@@ -1818,7 +1818,7 @@ def is_sorted(pda: pdarray) -> np.bool_:
     RuntimeError
         Raised if there's a server-side error thrown
     """
-    repMsg = generic_msg(cmd="reduction", args="{} {}".format("is_sorted", pda.name))
+    repMsg = generic_msg(cmd="reduction", args={"op": "is_sorted", "array": pda.name})
```

Comment:
Is the `.name` required here due to `Datetime` issues as well?

Reply:
Since that got merged, I will try it again.

Reply:
Removed `name` since the ticket adding Datetime parsing got merged